pkg/daemon/update.go

```diff
@@ -37,6 +37,8 @@ const (
 	coreUserName = "core"
 	// SSH Keys for user "core" will only be written at /home/core/.ssh
 	coreUserSSHPath = "/home/core/.ssh/"
+	// fipsCommand is the command to use when enabling or disabling FIPS
+	fipsCommand = "/usr/libexec/rhcos-tools/coreos-fips"
```

Comment:
Sort of a side comment but I recently was thinking that we probably should have taken the path of *replacing* `/usr/bin/fips-mode-setup`; we can totally do that now via [override layers](https://github.com/projectatomic/rpm-ostree/pull/1830).